openassessment/workflow/api.py

```diff
@@ -197,11 +197,18 @@ def update_from_assessments(submission_uuid, assessment_requirements, override_s
             relationship between submissions and workflows, so this uniquely
             identifies the `AssessmentWorkflow`.
         assessment_requirements (dict): Dictionary that currently looks like:
-            `{"peer": {"must_grade": <int>, "must_be_graded_by": <int>}}`
+            `{"peer": {
+                 "must_grade": <int>,
+                 "must_be_graded_by": <int>,
+                 "enable_flexible_grading": <bool>
+            }}`
             `must_grade` is the number of assessments a student must complete.
             `must_be_graded_by` is the number of assessments a submission must
             receive to be scored. `must_grade` should be greater than
             `must_be_graded_by` to ensure that everyone will get scored.
+            `enable_flexible_grading` looses the number of required peer
```

Comment:
I think the desired term is "loosens" as opposed to "looses"

Reply:
@jnlapierre yes. Updated.